packages/app-desktop/gulpfile.js

```diff
@@ -35,6 +36,13 @@ const tasks = {
 			);
 		},
 	},
+	bundleDefaultPlugins: {
+		fn: async () => {
+			const toolsDirectory = `${__dirname}/../tools`;
+			process.chdir(toolsDirectory);
```

Comment:
that shouldn't be here. The script itself should know from where directory it needs to run, so any change of directory should be in the script